primitives/src/constants.rs

```diff
@@ -39,32 +39,19 @@ pub mod currency {
 pub mod time {
 	use crate::{BlockNumber, Moment};
 
-	/// Since BABE is probabilistic this is the average expected block time that
-	/// we are targeting. Blocks will be produced at a minimum duration defined
-	/// by `SLOT_DURATION`, but some slots will not be allocated to any
-	/// authority and hence no block will be produced. We expect to have this
-	/// block time on average following the defined slot duration and the value
-	/// of `c` configured for BABE (where `1 - c` represents the probability of
-	/// a slot being empty).
-	/// This value is only used indirectly to define the unit constants below
-	/// that are expressed in blocks. The rest of the code should use
-	/// `SLOT_DURATION` instead (like the Timestamp pallet for calculating the
-	/// minimum period).
+	/// BLOCKS will be produced at a minimum duration defined by `SLOT_DURATION`.
+	/// `SLOT_DURATION` is picked up by `pallet_timestamp` which is in turn picked
+	/// up by `pallet_aura` to implement `fn slot_duration()`.
 	///
-	/// If using BABE with secondary slots (default) then all of the slots will
-	/// always be assigned, in which case `MILLISECS_PER_BLOCK` and
-	/// `SLOT_DURATION` should have the same value.
-	///
-	/// <https://research.web3.foundation/en/latest/polkadot/BABE/Babe/#6-practical-results>
-
-	pub const MILLISECS_PER_BLOCK: u64 = 12000;
+	/// Change this to adjust the block time.
+	pub const MILLISECS_PER_BLOCK: u64 = 6000;
+	pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
 
 	// Time is measured by number of blocks.
 	pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 	pub const HOURS: BlockNumber = MINUTES * 60;
```

Comment:
session length uses this constant, and that shouldn't probably change as it would break conversion from block number to session